server/app/attestation-service-ctl/readme.md

```diff
@@ -0,0 +1,129 @@
+# attestation-service-ctl
+
+Currently, `attestation-service-ctl` is a binary application written in Rust. It mainly provides the following functions:
+- Set/Get each TEE's Open Policy Engine(OPA) `Policy(.rego)` and `Reference Data(.json)` files. The `Reference Data(.json)` contents should come from such as Reference Value Provider Service(RVPS), but this item is not reflected in this implementation.
+- Restore each TEE's `Policy(.rego)` and `Reference Data(.json)` to default value.
+- Provide the Attestation Server's `attestation` endpoint testing functionality.
+
+## Supported TEEs
+
+Currently the `attestation-service-ctl` supports the following types of TEE:
+- sgx
+- tdx
+- sevsnp
+- sample: The TEE used to demo/test Attestation Server's functionalities.
+
+## Commands
+
+The `attestation-service-ctl` supports each TEE's `Policy(.rego)` and `Reference Data(.json)` configuration and testing subcommands, as the following:
+- get-policy
+- set-policy
+- get-reference-data
+- set-reference-data
+- restore-default-policy
+- restore-default-reference-data
+- test-attestation
+
```

Comment:
I'm going to be a PITA and I apologize in advance, but I think we want to have these commands a bit more organized. Something like that, to be discussed:

```
ctl policy set
ctl policy get
ctl policy restore
ctl reference-data set
ctl reference-data get
ctl reference-data restore
ctl test

```

With that approach we only have 3 top level commands (policy, reference-data and test) and we can add any sub-commands to any of them at any point in time without endlessly expanding our list of commands. I feel it makes for a nicer user experience.